esmvaltool/diag_scripts/shared/plot/xy_line.ncl

```diff
@@ -1520,45 +1520,45 @@ begin
 
   ; Select colors and other plotting attributes for multi-model mean
   if (diag_script_info@multi_model_mean .eq. "y") then
-    ; Project_style evaluates metadata of variable "models"
-    temp = models
+    ; Project_style evaluates metadata of variable "input_file_info"
+    temp = input_file_info
```

Comment:
For consistency with the other plot routines, I think we should use `dataset_info`, although the content is identical.